### PR TITLE
fixed missing translations

### PR DESCRIPTION
**Pull request by Fabio1988**
If you have furrther improvements let me know, I will implement it here. Is something missing yet?
## Description

added/fixed translations in en.json and de.json

## Motivation and Context

keeping translations up to date.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by SkOODaT** [comment]
your gonna need 
        .itemTroyDiskRainy,
https://github.com/RealDeviceMap/RealDeviceMap/blob/master/Sources/RealDeviceMap/Misc/POGOProtos_Inventory_Item_ItemId.swift
and its possible flos swift protos needs an update 
    case .itemTroyDiskRainy: return 505

**Comment by SkOODaT** [comment]
wow i missed a bunch too theirs more in index.js tho still lol

**Comment by Fabio1988** [comment]
I think I added on every place. But SwiftProtos doesn't contain the .itemTroyDiskRainy, it have to be updated in that repository of @123FLO321

File: Sources/RealDeviceMap/API/ApiRequestHandler.swift [hunk 1]
```diff
@@ -964,7 +964,7 @@ class ApiRequestHandler {
                 "type": pokestopOptionsString
             ])
 
-            for i in 1...4 {
+            for i in 1...5 {
                 let pokestopLure = Localizer.global.get(value: "filter_pokestop_lure_\(i)")
```
Comment (SkOODaT) [inline Sources/RealDeviceMap/API/ApiRequestHandler.swift:968]: missing 
```                </label>
                <label class="btn btn-sm btn-size select-button-new" data-id="\(i)"
                 data-type="pokestop-lure" data-info="huge">
                <input type="radio" name="options" id="show" autocomplete="off">\(hugeString)
                </label>```

Comment (SkOODaT) [reply Sources/RealDeviceMap/API/ApiRequestHandler.swift:968]: i think thats it, i looked over everything and it seems to be what i have im just not sure about index as its different then mine, i tested with a dummy lure in database too and it seemed to work, just need to actual real data to now

**Comment by Fabio1988** [comment]
depends on #207 , will adapt last changes when 207 is merged :)

File: Sources/RealDeviceMap/API/ApiRequestHandler.swift (hunk 1 above)
Comment (Fabio1988) [reply Sources/RealDeviceMap/API/ApiRequestHandler.swift:968]: this is only one row each. the change from `for i in 1...4` to `for i in 1...5` should be enough.
what you have duplicated is, that you added one more column for sizing information for the icon on map (small, normal, large, huge)

Comment (SkOODaT) [reply Sources/RealDeviceMap/API/ApiRequestHandler.swift:968]: thank you wasnt too sure how that worked, ill fix on mine :)

**Comment by SkOODaT** [comment]
only thing you need to adapt to new protos is 
 .itemTroyDiskRainy, ->  .troyDiskRainy,
 :)

**Comment by Fabio1988** [comment]
> only thing you need to adapt to new protos is
> .itemTroyDiskRainy, -> .troyDiskRainy,
> :)

I removed this from my changes, because it's already done in #207 :+1:

**Review by SkOODaT — DISMISSED** [review]

**Comment by Fabio1988** [comment]
I removed changes for rainy lure here. because it's not needed... we need those translations @123FLO321 , please merge... quest_57 is live.... for rainy lures I will create a separate PR.

**Pull request merged**